src/pudl/extract/phmsagas.py

```diff
@@ -52,6 +52,32 @@ def process_renamed(self, newdata: pd.DataFrame, page: str, **partition):
                     f"\n{to_drop}"
                 )
                 newdata = newdata.drop(columns=to_drop, errors="ignore")
+        # there is an annoying middling number of columns in phmsa raw data that are unnamed
+        # and have a smattering of random values. we want to drop these guys. but we are going
+        # to enumerate what we expect to need to
```

Comment:
It seems like this is the only special case that you ended up needing to deal with all the current errors, which is great! Do you want to finish this thought that got left off mid-sentence in the comment?

Reply:
ah yes lol my classic mid sentence walk away

Reply:
updated